docusaurus/website/pages/en/index.js

```diff
@@ -119,6 +125,26 @@ const Features = () => (
   </Block>
 );
 
+const GetStarted = props => (
+  <Block layout="twoColumn" background="light" {...props}>
+    {[
+      {
+        title: 'Get started coding in a matter of seconds!',
+        content: `With Create React App, you get to focus on **writing React, not boilerplate**.
+          All you need to do is run a command, install some dependencies, and decide what's for dinner.
+
+          npx create-react-app my-app
```

Comment:
Maybe we can tweak the CSS so the left hand side margin isn't so large in `code` blocks, what do you think?

Reply:
Sure I'll try to tune it down a bit :)

Reply:
Ah, that's actually indentation - I'll just remove the indentation and we should be good :+1:

Reply:
Cool, yeah it seemed not as large in the `.md` files. 

How did you generate https://create-react-app-stngeqjqos.now.sh/? When poking around, I followed [this](https://stackoverflow.com/questions/9153598/how-do-i-fetch-a-branch-on-someone-elses-fork-on-github) and pulled your branch locally.

Reply:
> How did you generate https://create-react-app-stngeqjqos.now.sh/? 
I use [now from zeit](https://zeit.co/now) - it's free and pretty easy to use. Once you've installed their CLI and logged in, you can do something like:

```sh
cd docusaurus/webpage
yarn build # to generate the html files
cd build/create-react-app
now
```

Reply:
Thanks! I'll have to check that out, and carbon too